driver/src/main/java/org/neo4j/driver/internal/async/UnmanagedTransaction.java

```diff
@@ -158,7 +158,13 @@ else if ( state == State.ROLLED_BACK )
 
     public boolean isOpen()
     {
-        return state != State.COMMITTED && state != State.ROLLED_BACK;
+        // TERMINATED is still considered open w.r.t to resources available
+        return state == State.ACTIVE || state == State.TERMINATED;
```

Comment:
That doesn't actually change anything (ACTIVE, TERMINATED) are the inverse of (COMMITED;ROLLED_BACK).

Reply:
That was intended to make it more readable (plus makes the comment easier to make sense of). Happy to roll it back.